### ISSUE TITLE
Awesome Windows Repository

### ISSUE THREAD

**Issue by rahulkapoor90** [sindresorhus/awesome#671]
when i started awesome windows repository i didn't knew how to write down the list properly but still i managed to gather all the resources and made a nice awesome windows repository and i did send a pull request here to add it here but instead it was closed because of less stars by sindresorhus.

but soon after that a guy with username [https://github.com/RiseLedger](RiseLedger) copied the whole thing i wrote and managed to get more 'stars' than me and you added him.

Is the one who wrote it originally is not entitled to be added here or is it just you can copy someone's else work and get more stars and go on sending pull request? Is this how the whole thing works?

My repository - [https://github.com/rahulkapoor90/awesome-windows](url)

RiseLedger - [https://github.com/RiseLedger/awesome-windows](url)

My pull request is here [https://github.com/sindresorhus/awesome/pull/326](url).

**Comment by davisonio** [comment]
> Is the one who wrote it originally is not entitled to be added here or is it just you can copy someone's else work and get more stars and go on sending pull request? Is this how the whole thing works?

Not at all. You should be the one who should be added to this list.

I see what the problem is from these commits:
- You originally created content here: September 26 2015 https://github.com/rahulkapoor90/awesome-windows/tree/51cbb5dd9748ab751d4f1aeb29c9c2e9360b72b4
- It was copied 4 days later by RiseLedger and looks very similar: September 30 2015 https://github.com/RiseLedger/awesome-windows/tree/b3b0ee2acf84f2b1c9d2f6d50d430552054077cc
- Also, as your content was licensed under CC-BY then RiseLedger should have credited...

@RiseLedger can you explain?

Else, I think the best thing to do moving forward is to add @rahulkapoor90's repository to this main list instead of @RiseLedger's considering the list that was created first with the original content.

**Comment by egeerardyn** [comment]
First of all, your #326 has **NOT** been closed due to "too little stars" as you mention. Instead, it's the typical reason https://github.com/sindresorhus/awesome/pull/326#issuecomment-141786349:

> Wait at least a couple of weeks after creating a list before submitting to this list to give it a chance to mature.

By the way, the other list was not accepted at first for the same reason. However, @RiseLedge took the time afterwards to get his list included, while you did not. I suspect that is the reason your list is not included and his one is.

Next to the evidence of @davisonio that shows that @RiseLedger based his list on @rahulkapoor90's work, I found a verbatim piece of overlap (not just "similar"):

>### Compression
>
>- [7-Zip](http://www.7-zip.org/) - Open source Windows utility for manipulating archives. Formats 7z, ZIP, GZIP, BZIP2 and TAR are supported fully, other formats can be unpacked.
>- [PeaZip](http://www.peazip.org/) - Archiver and file compressor. Extract 7Z CAB ISO RAR TAR ZIP archive files.
>- [WinRAR](http://www.rarlab.com/) -  A powerful archive manager. It can backup your data and reduce the size of email attachments, decompress RAR, ZIP and other files.

This can be tracked back to 
 - https://github.com/RiseLedger/awesome-windows/commit/ab5107450b7c367d5475dac0a08d56d7bd442bbe from September 28, 2015 (i.e. in @RiseLedger's original PR #341 from Oct 1, 2015), and
 - https://github.com/rahulkapoor90/awesome-windows/commit/439b3d906b22aa2a9b9915923ef50832b8850b8d from December 23, 2015 (i.e. made after your PR #326 from Sep 20, 2015 ánd after his PR).

Note also that the latter commit contains more verbatim fragments from @RiseLedger's repository without any attribution and a snarky comment on top.
_I.e. evidence that @rahulkapoor90 also plagiarized @RiseLedger's work._

Personally, I think that @rahulkapoor90's list needs a lot of work still:
 - some links are wrong (e.g. Textual 5 (Mac OS X application) links to mIRC (WindowS)),
 - some programs do not even exist for Windows (e.g. xScope, Pixelmator, ...) 

**TL;DR:** I think the best course of action is to decide among each other whether or not to collaborate (merge both lists, decide to maintain it together, ...). But I guess the best place for that is to open an issue either in your repository mentioning him or an issue in his repository.

Regardless of cross-plagiarism, I would advise against blindly changing the reference without further review of the quality of both lists and explanations of both involved.

**Comment by RiseLedger** [comment]
Hi @rahulkapoor90 . As you mention seems that your list was created earlier then my list. I was not aware that such list exist, that's why I created it in first place. 

All the apps that I added in [initial commits](https://github.com/RiseLedger/awesome-windows/blob/b3b0ee2acf84f2b1c9d2f6d50d430552054077cc/readme.md) - is application that I use daily on the windows machine for years, and know that they are good, and for help of creating categories and decided which apps to include in the list, I was following [ninite website](https://ninite.com/).

I see however some overlaps at the beginning in **Terminal** and **Editors** categories.

Like @egeerardyn mention it will be better to create an issue in your or my repository, and decide which path we will take next.

**Comment by rahulkapoor90** [comment]
@egeerardyn i accept that i also included some of the things that @RiseLedger added as i was angry at that time and didn't knew what to do and clearly made a rookie mistake **but**  i also thought i would start contributing daily to my repository and people would somehow recognize that mine is the original one but it all went the other way around.

and i even added this line - 

> Some users have made their own repositories by copying content from this repository, i request all of you to please fork only this original repository and add/suggest/submit changes.

**Comment by egeerardyn** [comment]
@rahulkapoor90: I understand why where angry/disappointed that somebody else did something very similar to your work and gets the credit.

However, in such cases it is often most productive to assume no malicious intent and just contact the other author, see what has happened, offer to collaborate and share the workload (because maintaining a good list is quite some work, as you have probably noticed) and also the credit.

What throws me off and looks like copying is the _identical_ wording at the top: `A curated list of awesome applications, softwares, tools and shiny things for Windows` in both repositories since the beginning.

While this could point towards @RiseLedger copying @rahulkapoor90 , I think the explanation is much simpler: both @rahulkapoor90 and @RiseLedger may have used the [same repository](https://github.com/iCHAIT/awesome-osx) as a starting point and inspiration: it also contains `A curated list of awesome applications, softwares, tools and shiny things for OS X`.

Technically, since that is a CC-BY repo, it should have been mentioned by both.
So I honestly believe that this is also a case of just two people working on something very similar at the same time, without any intent of profiting of the other's work.

I am glad to see https://github.com/RiseLedger/awesome-windows/issues/29 where I think further discussion is most appropriate.

**Comment by sindresorhus** [comment]
Thanks excellent handling of this @egeerardyn :)

**Comment by egeerardyn** [comment]
@sindresorhus You're welcome. 
I think both @RiseLedger and @rahulkapoor90 have made huge improvements to their joint list since a week ago. So I hope we can treat their [new list](https://github.com/Awesome-Windows/Awesome) as just a replacement of the old one when both are ready to fully release it.
